common/baselines/librispeech/ls960/gmm/baseline_args.py

```diff
@@ -0,0 +1,457 @@
+"""
+Definition of all changeable pipeline parameters of the GMM-RASR-Pipeline
+
+The parameters defined here are based on the past experience with training LibriSpeech-100h and 960h models,
+but no excessive tuning has been done.
+"""
+from i6_core.features.filterbank import filter_width_from_channels
+from i6_core import cart
+
+from i6_experiments.common.setups.rasr import util
+from i6_experiments.common.datasets.librispeech.cart import (
+    CartQuestionsWithStress,
+    CartQuestionsWithoutStress,
+)
+from i6_experiments.common.baselines.librispeech.default_tools import SCTK_BINARY_PATH
+
+
+def get_init_args():
+    dc_detection = False
+    samples_options = {
+        "audio_format": "wav",
+        "dc_detection": dc_detection,
+    }
+
+    am_args = {
+        "state_tying": "monophone",
+        "states_per_phone": 3,
+        "state_repetitions": 1,
+        "across_word_model": True,
+        "early_recombination": False,
+        "tdp_scale": 1.0,
+        "tdp_transition": (3.0, 0.0, "infinity", 0.0),  # loop, forward, skip, exit
+        "tdp_silence": (0.0, 3.0, "infinity", 20.0),
+        "tying_type": "global",
+        "nonword_phones": "",
+        "tdp_nonword": (
+            0.0,
+            3.0,
+            "infinity",
+            6.0,
+        ),  # only used when tying_type = global-and-nonword
```

Comment:
As these influence the hash but do not have an effect, better remove or comment them out. 
(Or change the tying type to "global-and-nonword" and provide some "nonword_phones")

If it was used, I would for `tdp_nonword[3]` (i.e. exit tdp) replace 6.0 with 21.0 to make it higher than silence.exit so that if in doubt the TDP select silence over nonwords